accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -828,6 +829,9 @@ type)*.  Exhaustiveness analysis will treat such patterns as if they had
 been an object pattern matching the extension type erasure of `V` (defined
 below).
 
+An extension type `V` is always-exhaustive if and only if its instantiated 
+representation type is always-exhaustive.
```

Comment:
Could we add examples?

> _For example a type `TriBool` defined as `extension type TriBool(bool? _) {}`, is always-exhaustive because the representation type, `bool?`, is.
> For a declaration like `extension type Pair<S, T>((S, T) _) {}`, the type `Pair<SomeEnum, TriBool>` is always-exhaustive because `(SomeEnum, TriBool)` is, and `Pair<String, bool>` is not always-exhaustive because `(String, bool)` isn't._

Reply:
Nice! Done.